src/web/app/components/question-types/question-edit-details-form/mcq-field/mcq-field.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { StatusMessageService } from '../../../../../services/status-message.service';
+
+/**
+ * The input field to specify options to choose from.
+ */
+@Component({
+  selector: 'tm-mcq-field',
+  templateUrl: './mcq-field.component.html',
+  styleUrls: ['./mcq-field.component.scss'],
+})
+export class McqFieldComponent {
+
+  @Input()
+  isEditable: boolean = false;
+
+  @Input()
+  numberOfMcqChoices: number = 1;
+
+  @Input()
+  text: string = '';
+
+  @Input()
+  index: number = 0;
+
+  @Output()
+  elementDeleted: EventEmitter<any> = new EventEmitter();
+
+  @Output()
+  mcqText: EventEmitter<any> = new EventEmitter();
+
+  constructor(private statusMessageService: StatusMessageService) {}
+
+  /**
+   * Deletes a Mcq.
+   */
+  deleteMcqOption(): void {
+    if (this.numberOfMcqChoices > 2) {
+      this.elementDeleted.emit(this.index);
+    } else {
+      this.statusMessageService.showErrorMessage('There must be at least two Mcq options.');
```

Comment:
`MCQ` or `Mcq`?
Also, you should standardize the casing. You should not mix up `mcq`, `MCQ`, `Mcq` unless the situation calls, e.g. for variable name `mcq*` or `*Mcq*` is appropriate, but everything else should be standardized.